app/rww/ldp/WebClient.scala

```diff
@@ -154,16 +151,19 @@ class WSClient[Rdf <: RDF](readerSelector: ReaderSelector[Rdf, Try], rdfWriter:
     val maybeMimeType = maybeContentType.flatMap(ct => MimeType.parse(ct))
     maybeMimeType.map {
       _ match {
-        case MimeType("text", "plain", _) => AllRdfMimeTypes
-        case MimeType("application", "xml", _) => Set(MimeType("application", "xml"), MimeType("application", "rdf+xml"))
-        case MimeType("text", "xml", _) => Set(MimeType("text", "xml"), MimeType("application", "rdf+xml"))
-        case mimeT => Set(mimeT)
+        case MimeType("text", "plain", _)      => AllRdfMimeTypes
+        case MimeType("application", "xml", _) => Set(MimeType("application", "xml"), MimeType("application", "rdf+xml"), MimeType("text", "rdf+xml"))
```

Comment:
I think here the aim should be to return a set of official mime types for which we have a parser. We don't have one for `text/xml`